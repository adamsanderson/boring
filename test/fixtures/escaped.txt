test_result.log:
[1;31mFailure:[22m
test_comments_generate_change_reports(ActivityStreamTest)[0m