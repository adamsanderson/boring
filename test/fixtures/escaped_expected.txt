test_result.log:
Failure:
test_comments_generate_change_reports(ActivityStreamTest)